internal/storage/bucket_handle_test.go

```diff
@@ -405,3 +405,46 @@ func (t *BucketHandleTest) TestListObjectMethodForMaxResult() {
 	AssertEq(TestObjectSubRootFolderName, twoObj.Objects[1].Name)
 	AssertEq(nil, twoObj.CollapsedRuns)
 }
+
+func (t *BucketHandleTest) TestUpdateObjectMethodWithValidObject() {
+	var ContentType string = "ContentType"
+	var ContentEncoding string = "ContentEncoding"
+	var ContentLanguage string = "ContentLanguage"
+	var CacheControl string = "CacheControl"
+
+	_, err := t.bucketHandle.UpdateObject(context.Background(),
+		&gcs.UpdateObjectRequest{
```

Comment:
what are the object contents before calling updateObject. Can we do a check on that as well.

Reply:
commented